### DISCUSSION TITLE
someone please, make space bar restart from a selected point.

### DISCUSSION THREAD

**Discussion by ghost** [LMMS/lmms#6713] [Support request] [answered]
sometimes when i make very tiny small changes to the melody, i have to start all the way from the beginning of the track when i press space (i can move it but i would have to move it each time whch is inconvenient.).. am i missing something? since it's much harder for me that way if there is a solution already, i suggest it should be more intuitive for the design of the shortcuts.

**Comment by sakertooth** [comment, marked as answer]
If you look at one of the options in the Song-Editor, there should be a choice to change this behavior. The symbols are |<<, <<, and >>.

**Reply by ghost** [reply]
thank you, i've been struggling with it for months.
